Cargo.toml

```diff
@@ -34,12 +34,12 @@ inspect = ["bevy-inspector-egui"]
 serde = ["dep:serde"]
 
 [dependencies]
-bevy = { version = "0.14", default-features = false, features = [
+bevy = { version = "0.15", features = [
```

Comment:
Is there any reason to bring all dependencies by default? I removed it because being a library we really want it to be as light as possible

Reply:
Good call- deps updated